docs/content/manual/pre-release/cluster-restore/system-upgrade-with-restore.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Longhorn system upgrade with restore
+---
+
+Notice that the behaviors will be different if the cluster node roles are different. e.g., A cluster contains 1 dedicated master node + 3 worker node is different from a cluster contains 3 nodes which are both master and worker.
+This test may need to be validated for both kind of cluster.
+
+1. Install Longhorn system. Deploy one more compatible engine image.
+2. Deploy some workloads using Longhorn volumes then write some data.
+3. Create a cluster snapshot via Rancher.
+4. Upgrade the Longhorn system to a newer version. Then modify the settings or node configs (especially the configs introduced in the new version).
```

Comment:
Should be validate the new volumes created with the upgraded engine at this step should disappear after the restore and doesn't create any problem?

Reply:
I was thinking this step may be duplicate. If it's better to have, I will add it then.